website/routes.py

```diff
@@ -173,6 +173,8 @@ def make_url_map(app):
 
     process_rules(app, [
 
+        Rule('/home/', 'get', website_views.home, OsfWebRenderer('home.mako')),
```

Comment:
What is this route for? How is different than what we have?

Sent and possibly dictated from my iPhone

> On Dec 15, 2015, at 10:35 AM, Steven Loria <notifications@github.com> wrote:
> 
> In website/routes.py:
> 
> > @@ -173,6 +173,8 @@ def make_url_map(app):
> >  
> >      process_rules(app, [
> >  
> > +        Rule('/home/', 'get', website_views.home, OsfWebRenderer('home.mako')),
> Ping @JeffSpies for approval of this new route.
> 
> —
> Reply to this email directly or view it on GitHub.
>